sdk/contentsafety/azure-ai-contentsafety/samples/sample_manage_blocklist.py

```diff
@@ -0,0 +1,189 @@
+    try:
+        response = client.list_text_blocklist_items(blocklist_name=name)
+        return list(response)
+    except HttpResponseError as e:
+        print("List block items failed.")
+        print("Error code: {}".format(e.error.code))
+        print("Error message: {}".format(e.error.message))
+        return None
+    except Exception as e:
+        print(e)
+        return None
+
+def remove_block_items(name, items):
+    request = RemoveBlockItemsOptions(block_item_ids=[i.block_item_id for i in items])
+    try:
+        client.remove_block_items(blocklist_name=name, body=request)
+        return True
+    except HttpResponseError as e:
+        print("Remove block items failed.")
+        print("Error code: {}".format(e.error.code))
+        print("Error message: {}".format(e.error.message))
+        return False
+    except Exception as e:
+        print(e)
+        return False
+
+def add_block_items(name, items):
+    block_items = [TextBlockItemInfo(text=i) for i in items]
+    try:
+        response = client.add_block_items(
+            blocklist_name=name,
+            body=AddBlockItemsOptions(block_items=block_items),
+        )
+    except HttpResponseError as e:
+        print("Add block items failed.")
+        print("Error code: {}".format(e.error.code))
+        print("Error message: {}".format(e.error.message))
+        return None
+
+    except Exception as e:
+        print(e)
+        return None
+
+    return response.value
+
+def get_block_item(name, item_id):
+    try:
+        return client.get_text_blocklist_item(blocklist_name=name, block_item_id=item_id)
+    except HttpResponseError as e:
+        print("Get block item failed.")
+        print("Error code: {}".format(e.error.code))
+        print("Error message: {}".format(e.error.message))
+        return None
+    except Exception as e:
+        print(e)
+        return None
+
+def analyze_text_with_blocklists(name, text):
+    try:
+        response = client.analyze_text(
+            AnalyzeTextOptions(text=text, blocklist_names=[name], break_by_blocklists=False)
+        )
+    except HttpResponseError as e:
+        print("Analyze text failed.")
+        print("Error code: {}".format(e.error.code))
+        print("Error message: {}".format(e.error.message))
+        return None
+    except Exception as e:
+        print(e)
+        return None
+
+    return response.blocklists_match_results
+
+def delete_blocklist(name):
+    try:
+        client.delete_text_blocklist(blocklist_name=name)
+        return True
+    except HttpResponseError as e:
+        print("Delete blocklist failed.")
+        print("Error code: {}".format(e.error.code))
+        print("Error message: {}".format(e.error.message))
+        return False
+    except Exception as e:
+        print(e)
+        return False
+
+
+if __name__ == "__main__":
+    blocklist_name = "Test Blocklist"
+    blocklist_description = "Test blocklist management."
+
+    # list blocklists
```

Comment:
I wouldn't include any code in `__main__` besides calling the sample functions. If you make the sample functions focus more on the response body, the rest of the code from 142 down is unnecessary

Reply:
Moved usage to functions and keep the __main__ clean with only function calls.